ProfessorAssessor/View Controllers/ProfessorViewController.m

```diff
@@ -9,7 +23,141 @@ @implementation ProfessorViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
 
-    // TODO: fetch reviews
+    [self setProfessorDetails];
+
+    [self fetchCoursesAndReviews];
+
+    [self setUpRefreshControl];
+}
+
+- (void)setProfessorDetails {
+    self.professorName.text = self.professor.name;
+    self.departmentName.text = self.professor.departmentName;
+    self.averageRating.value = [self.professor.averageRating doubleValue];
+}
+
+- (void)fetchCoursesAndReviews {
+    self.courses = [self sortedCourses];
+    self.selectedCourses = [NSMutableArray new];
+
+    [self displayCourseTags];
+
+    [self fetchReviews];
+}
+
+- (void)fetchReviews {
+    __weak typeof(self) weakSelf = self;
```

Comment:
Does this need to show an activity indicator? Should it prevent UI interaction during the fetch?